luda/editor/client/src/timeline/clip/renderClip.ts

```diff
@@ -0,0 +1,42 @@
+import { Render } from "namui";
+import { isCameraClip, isSubtitleClip } from "../../clipTypeGuard";
+import { Clip } from "../../type";
+import { CameraTrackSash } from "../cameraTrack/CameraTrackSash";
+import { TimelineState } from "../type";
+import { ClipComponent } from "./ClipComponent";
+import { SubtitleClipComponent } from "./SubtitleClipComponent";
+
+export const renderClip: Render<
+  {
+    timelineState: TimelineState;
+    clip: Clip;
+  },
+  {
+    height: number;
+    maxRight: number;
+  }
+> = (state, props) => {
+  const { clip } = state;
+
+  if (isCameraClip(clip)) {
+    console.log("camera");
+    return ClipComponent(state, {
+      ...props,
+      sashComponent: CameraTrackSash,
+    });
+  }
+
+  if (isSubtitleClip(clip)) {
+    return SubtitleClipComponent(
+      {
+        ...state,
+        clip,
+      },
+      props,
+    );
+  }
+
+  throw new Error(
+    "No rendering implementation for this clip type. Please implement it.",
```

Comment:
I think it is ok because we only have 2 track, but I recommend returning `ClipComponent`. with sashComponent `luda/editor/client/src/timeline/clip/Sash.ts`.

Reply:
That looks better. Fixed
with https://github.com/NamseEnt/namseent/pull/18/commits/1b73b02f3bdf4f1bba8a580d29d8062d1ddc4457